mjx/mujoco/mjx/testspeed.py

```diff
@@ -50,7 +50,12 @@ def _main(argv: Sequence[str]):
   path = epath.resource_path('mujoco.mjx') / 'test_data'
   path = _BASE_PATH.value or path
   f = epath.Path(path) / _MJCF.value
-  m = mujoco.MjModel.from_xml_path(f.as_posix())
+  if f.suffix == '.xml':
```

Comment:
This might be more appropriate:

```
if f.suffix == 'mjb':
  m = mujoco.MjModel.from_binary_path(f.as_posix())
else:
  m = mujoco.MjModel.from_xml_path(f.as_posix())
```

Since it [1] defaults to XML unless the file extension is mjb and [2] lets the MuJoCo compiler determine if the XML format is unknown.